canonical/lxd-ui
# Fix storage naming

## Done

- rename storage/-s to storage-pools
- rename menu entry to "storage"
- rename heading to "storage pools" on the list page
- add breadcrumb to storage pool detail page

Fixes #332

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described in the [Readme](https://github.com/canonical/lxd-ui#setting-up-for-development).
2. Perform the following QA steps:
    - check storage pages
